assets/js/lib/settings.js

```diff
@@ -66,6 +66,11 @@ class SettingsStore {
   _loadSettings() {
     const settings = load(SETTINGS_KEY);
 
+    // Rewrite settings for backward compatibility
+    if (settings.editor_theme === "highContrast") {
+      settings.editor_theme = "default";
```

Comment:
```suggestion
    if (settings.editor_theme === "light") {
      settings.editor_theme = "light";
```

This way we don't have to worry about previous values. Thoughts?

Reply:
wdym? We need to make sure to somehow handle `"highContrast"` value that users already have in local storage, and we settled on rewriting it to the default theme, right?

Reply:
Sorry, that snippet was poor. I was thinking about something like this:

```suggestion
    if (settings.editor_theme === "light") {
      this._settings.editor_theme = "light";
```

So instead of merging at the end, you explicitly copy valid values, this way you don't have to worry about invalid ones.

I am fine either way though :)

Reply:
Sounds good, but I think we should check all valid values for completeness, otherwise it can break in the future. Updated :)